.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+  // only enable if you are using a virtual environment
+  "python.pythonPath": ".venv/bin/python",
```

Comment:
I think on Windows the path is `.venv/Scripts/python`. Can anyone confirm?

Reply:
how annoying. no clue if that is correct